CHANGELOG.md

```diff
@@ -1,85 +1,115 @@
 Changelog
 =========
 
+unreleased (YYYY-MM-DD)
```

Comment:
Should this file be updated in a separate PR?

Reply:
I can remove the CHANGELOG update from this PR and publish a new one for doing it, but we do have users using the pre-commit-hooks from master and it would be nice if we do communicate what changes are they using ;)

Reply:
Moved in #31